README.md

```diff
@@ -1 +1,83 @@
-# CytoPipe
\ No newline at end of file
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [CytoPipe](#cytopipe)
+- [Formatting](#formatting)
+  - [smkfmt](#smkfmt)
+- [Testing](#testing)
+  - [Dry Runs](#dry-runs)
+
+# CytoPipe
```

Comment:
This should be line 1!

We should also add a quick description of the package.

Maybe something like:

"Reproducible pipelines for processing high-dimensional systems morphology data with snakemake 🐍 "

Reply:
TOC can go after this short description :)

Reply:
fixed! : https://github.com/WayScience/CytoPipe/pull/5/commits/7564f7a4b657c4dd35ce6e0d8705d305845f0926